Contribute/file-naming-conventions.md

```diff
@@ -4,69 +4,68 @@ versionFrom: 7.0.0
 
-But for documentation that applies to a range of versions, to make this as obvious as possible from the filename we will use vpost and vpre in the filename to indicate this eg:
+For documentation that applies to a range of versions, we will use `vpost` and `vpre` in the filename to indicate this. E.g.:
 
-`flexible-vpost-7.6.md` would contain the documentation used after v7.6 but before the next documentation version
-and `flexible-vpre-7.3.md` would contain the documentation for versions before v7.3.
+`index-vpost-7.6.md` would contain documentation to be used after version 7.6 but before the next documentation version
+and `index-vpre-7.3.md` would contain the documentation for versions before version 7.3.
 
 The `vpost` and `vpre` notations are not used to render to the user.
 
 For SEO reasons it is not necessary to change a file name when a feature becomes obsolete.  
 
 ## Adding meta data
 
-It is the [YAML meta data](adding-metadata.md) in the document itself, that will be used as the point of truth for when a (semver) version applies from and to.
-The YAML will be added to an examine index, along with the filename and is used later on for searching on (major) version.  Or to show the information to the user.
+It is the [YAML meta data](adding-metadata.md) in the document itself, that will be used as the "point of truth" for when a version applies from and to.
+The YAML will be added to an examine index, along with the filename and is used later on for searching on (major) version, or to show the information to the user.
 
-For versioning we use 2 YAML attributes:
+For versioning we use 3 YAML attributes:
 
-1. optional `versionFrom` to indicate a start version
-2. optional `versionTo` to indicate which version the support ended.
-
-Only the current version of the document can have both `versionFrom` and `versionTo` missing.  Otherwise at least one of the should be filled in.
+1. `versionFrom` to indicate a start version
+2. `versionTo` to indicate which version the support ended
+3. `versionRemoved` to indicate which in which version the feature described in the article was removed
 
 ## Discovering other pages
 
 Alternative pages for documentation will be discovered by searching the examine index for other files beginning with the same name e.g.
 
 If we are on the page
 
-    /Documentation/Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md
+    /Documentation/Getting-Started/Setup/Server-Setup/Load-Balancing/ContentService-Events.md
 
 we will search for any documents in the index beginning with
 
-     /Documentation/Getting-Started/Setup/Server-Setup/Load-Balancing/flexible
+     /Documentation/Getting-Started/Setup/Server-Setup/Load-Balancing/ContentService-Events
 
 which will return
 
-    /Documentation/Getting-Started/Setup/Server-Setup/Load-Balancing/flexible-vpre-v7.3.md etc
+    /Documentation/Getting-Started/Setup/Server-Setup/Load-Balancing/ContentService-Events-vpre-v7.3.md
```

Comment:
```suggestion
    /Documentation/Getting-Started/Setup/Server-Setup/Load-Balancing/ContentService-Events-vpre7.3.md
```

The versioning thing matches against `-v` in the filename, so having it there twice will definitely not work :)